### DISCUSSION TITLE
Disable new cmdline source

### DISCUSSION THREAD

**Discussion by dpetka2001** [saghen/blink.cmp#512] [Q&A] [answered]
Hey, how can I disable the newly default `cmdline` source and revert back to default Neovim `cmdline` completion?

**Comment by saghen** [comment, marked as answer]
`sources.cmdline = {}` should do it

**Reply by dpetka2001** [reply]
I've already tried that and it doesn't work in my case. Can you verify on your end that it works for you? I don't get any completion with this.

**Reply by saghen** [reply]
Whoops, the keymaps were still getting applied. Just pushed a fix for that

**Reply by dpetka2001** [reply]
Thank you very much for your fast response. Works as expected with latest update. :smile:
